crates/solver/src/settlement_access_list.rs

```diff
@@ -76,16 +100,63 @@ impl TenderlyApi {
     }
 }
 
+#[async_trait::async_trait]
+impl AccessListEstimating for TenderlyApi {
+    async fn estimate_access_lists(
+        &self,
+        web3: &Web3,
+        network_id: String,
```

Comment:
Would it make sense to remove these two parametes from the function and store them into the struct implemeneting the trait instead. To me it makes sense because they should be constant so there is no need to dynamically adjust what is passed.

Reply:
Fixed.